assignments/objects.js

```diff
@@ -19,26 +19,77 @@ let example = {
 
 // Write your intern objects here:
 
+let interns = [
```

Comment:
Is there a particular reason that you decided to put these objects in an array? This solution is definitely valid and I like that you thought about it this way. Another way we could have gone about it is to just create the objects and ultimately I think thats what the assignment expected based on the example object. The example had an individual object that was named and defined with a let. So for example:

let mitzi = {
"id": 1,
"name": "Mitzi",
"email": "mmelloy0@psu.edu",
"gender": "F"
}```
We could then access each object later by simply referencing mitzi._insertKeyFromObjectHere_